### DISCUSSION TITLE
Lots of us run Termux for work and can't have third-party-repos, can you upload new Termux versions to Goog Stor?

### DISCUSSION THREAD

**Discussion by ETERNALBLUEbullrun** [termux/termux-app#3615] [General]
Such a usful program for a lot of us!
Sad that Terminal IDE's gone, now just Termux stands out as a good terminal emu for AARCH.
Could funds allow to upload new versions?

**Comment by twaik** [comment]
**Termux on Google Play Store is OUTDATED**🪦
Termux and its plugins are no longer updated on the Google Play Store. This is due to API requirements that introduce new restrictions, which break Termux's basic functionality.
Please switch to the F-Droid version!
Direct link: <https://f-droid.org/en/packages/com.termux>
**Before** you install it, You **must** __fully uninstall__ current Termux and all plugins. If you don't, you will get a cryptic `app not installed` error!
(that means uninstalling the main app, as well as Termux:API, :Styling, :Widget, :Task, :Float, :Boot)
If you need help with backing up your files:
<https://wiki.termux.com/wiki/Backing_up_Termux>

**Comment by agnostic-apollo** [comment]
Read https://github.com/termux/termux-app#google-play-store-deprecated

Funds may help build an android 10 compatible termux variant, but not for getting us access to playstore account.

**Comment by ETERNALBLUEbullrun** [comment]
"RESOLVED"? Don't you mean "WONTFIX"? Problem = work won't allow 3rd-party-sources; solution can't be "install from 3rd-party-sources."
https://www.quora.com/unanswered/How-do-you-start-a-class-action-against-Google-for-Tivoization-of-Linux-Android-10-requires-rooting-just-for-terminal-emulation-to-run

**Reply by agnostic-apollo** [reply]
There is an alternate that can work, but requires lot of work and time.

https://github.com/termux/termux-app/issues/2155#issuecomment-1636732850
